visual-monitor/test/tests.js

```diff
@@ -61,12 +50,23 @@ describe('Visual monitor testing', function() {
   it('should show the home page',function(done) {
     client
       .url(baseUrl)
+      .execute(function() {
+        jQuery('body').css('background', 'none');
+      })
       .webdrivercss(testName + '.homepage', {
         name: '1',
-        exclude: [],
-        remove: [],
-        hide: [],
-        screenWidth: selectedCaps == 'chrome' ? [640, 960, 1200] : undefined,
+        exclude:
+          [
+            // Teachers and students.
+            '.hom_col .ttm_image',
+            '.hom_col .ttm_text',
+          ],
+        hide:
+          [
+            // Sidebar.
+            '.hom_col .hom_skills',
+          ],
+        screenWidth: selectedCaps == 'chrome' ? [1200] : undefined,
```

Comment:
lets use 960 by default instead of the 1200 (just looks better in the screenshots I do later from within shoov)